queue_services/entity-emailer/requirements.txt

```diff
@@ -37,7 +37,7 @@ isoduration==20.11.0
 itsdangerous==1.1.0
 Jinja2==2.11.3
 jsonpointer==2.3
-jsonschema==4.17.3
+jsonschema==4.19
 launchdarkly-server-sdk==7.1.0
```

Comment:
Bumped `attrs` and `jsonchema` versions to work with latest `business-schemas` version